data/quarg outfits.txt

```diff
@@ -39,6 +39,7 @@ outfit "Quarg Skylance"
 	weapon
 		sprite "projectile/skylance"
 			"frame rate" 1
+		"hardpoint sprite" "hardpoint/skylance"
```

Comment:
Note that we need to specify the `"hardpoint offset"` as well. 9 should be good.

Reply:
ahh okay

Reply:
fixed